xbmc/guilib/GUIAudioManager.cpp

```diff
@@ -53,6 +53,10 @@ void CGUIAudioManager::OnSettingChanged(const CSetting *setting)
   const std::string &settingId = setting->GetId();
   if (settingId == CSettings::SETTING_LOOKANDFEEL_SOUNDSKIN)
   {
+    //Migrate "OFF" setting
+    if (((CSettingString*)setting)->GetValue() == "OFF")
```

Comment:
You could write a setting update handler (see `OnSettingUpdate()`) to do this when the settings are loaded during startup.